source/user-manual/reference/ossec-conf/email_alerts.rst

```diff
@@ -91,13 +91,15 @@ format
 
 This specifies the email format.
 
-+--------------------+----------------------------------------------------+
-| **Default value**  | full                                               |
-+--------------------+------+---------------------------------------------+
-| **Allowed values** | full | Send normal emails.                         |
-+                    +------+---------------------------------------------+
-|                    | sms  | Use a compact format more suitable for SMS. |
-+--------------------+------+---------------------------------------------+
++--------------------+-------------------------------------------------------+
+| **Default value**  | full                                                  |
++--------------------+---------+---------------------------------------------+
+| **Allowed values** | default | Same as full.                               |
```

Comment:
Repeated label.

Reply:
How should I note that `default` means the same as `full` without mentioning the name `full` ?

I suppose I could copy the description, instead.  I will try that.